app/views/core/RootView.coffee

```diff
@@ -201,16 +201,23 @@ module.exports = class RootView extends CocoView
       #console.log 'Saved language:', newLang
 
   isOldBrowser: ->
-    if features.china and $.browser
+    if $.browser
       return true if not ($.browser.webkit or $.browser.mozilla or $.browser.msedge)
       majorVersion = $.browser.versionNumber
       return true if $.browser.mozilla && majorVersion < 25
-      return true if $.browser.chrome && majorVersion < 72  # forbid some chinese browser
+      return true if $.browser.chrome && majorVersion < 31  # Noticed Gems in the Deep not loading with 30
       return true if $.browser.safari && majorVersion < 6  # 6 might have problems with Aether, or maybe just old minors of 6: https://errorception.com/projects/51a79585ee207206390002a2/errors/547a202e1ead63ba4e4ac9fd
     else
       console.warn 'no more jquery browser version...'
     return false
 
+  isChinaOldBrowser: ->
+    if features.china && $.browser
+      majorVersion = $.browser.versionNumber
+      return true if $.browser.chrome && majorVersion < 72  # forbid some chinese browser
```

Comment:
@nwinter is it OK to limit the majorVersion to 72 for global users?

Reply:
That's probably a good idea; that's about a year old now. I'm sure 31 is too low a bar by this point!